src/libraries/System.Private.CoreLib/src/System/Diagnostics/StackTrace.cs

```diff
@@ -201,8 +201,11 @@ internal string ToString(TraceFormat traceFormat)
 
         internal void ToString(TraceFormat traceFormat, StringBuilder sb)
         {
-            string word_At = SR.Word_At;
-            string inFileLineNum = SR.StackTrace_InFileLineNumber;
+            // Passing a default string for "at" in case we are using Resource Keys instead of resources
```

Comment:
> in case we are using Resource Keys instead of resources

This wording may not make sense to someone new to the code.

```suggestion
            // Passing a default string for "at" in case SR.UsingResourceKeys() is true
```